content/en/blog/gitops-your-falco-rules/index.md

```diff
@@ -0,0 +1,817 @@
+...
+```
+
+Translated into our Helm chart `values.yaml`, the fields to modify would look like:
+
+```
+...
+falcoctl:
+  ...
+  config:
+    ...
+    artifact:
+      ...
+      follow:
+        refs:
+          - falco-rules:0
+          - ghcr.io/vjjmiras/cool-falco-ruleset/custom-rules:main  <-- Add this line
+        every: 2m                                                  <-- Modify this line
+        ...
+```
+
+Followed by Helm upgrade command:
+
+```
+$ helm upgrade falco -n falco --set tty=true falcosecurity/falco
+```
+
+Once we have updated those changes, we can wait for the new Pods with the current configuration:
+
+```
+$ kubectl -n falco get pods -w
+NAME          READY   STATUS     RESTARTS   AGE
+falco-np6hj   0/2     Init:1/2   0          4s
+falco-np6hj   0/2     PodInitializing   0          6s
+falco-np6hj   1/2     Running           0          7s
+falco-np6hj   2/2     Running           0          45s
+```
+
+Once the Pod is running, the logs from the sidecar container should tell us if all went as expected:
+
+```
+$ kubectl -n falco logs falco-np6hj -c falcoctl-artifact-follow
+INFO: Retrieving versions from Falco (timeout 2m0s) ...
+INFO: Successfully retrieved versions from Falco ...
+INFO: Reading all configured index files from "/root/.config/falcoctl/indexes.yaml"
+WARN: No configured index. Consider to configure one using the 'index add' command.
+INFO: Creating follower for "ghcr.io/vjjmiras/cool-falco-rules/custom-rules:main", with check every 2m0s
+```
+
+Our rules artifact is being followed now, it'll check every 2 minutes, so we can observe the content of the rules directory and the rules file:
+
+```
+$ kubectl -n falco exec falco-np6hj -c falco -it -- ls -l /etc/falco/
+
+-rw------- 1 root root     43 Apr 14 17:11 custom_falco_rules.yaml
+-rw-r--r-- 1 root root   1626 Apr 14 17:11 falco.yaml
+-rw-r--r-- 1 root root 147885 Apr 14 17:11 falco_rules.yaml
+
+$ kubectl -n falco exec falco-np6hj -c falco -it -- cat
+    /etc/falco/custom_falco_rules.yaml
+
+- list: falco_binaries
+  items: [falcoctl]
+```
+
+To test the workflow, as we did before, we can update the rules file locally, commit and push it to GitHub, and see if `falcoctl` updates it the same way it did in the previous section.
+
+After some minutes, this should be the content of the rules file inside the new container:
+
+```
+$ kubectl -n falco exec falco-np6hj -c falco -it -- \
+  cat /etc/falco/custom_falco_rules.yaml
+
+- list: falco_binaries
+  items: [falcoctl]
+
+- macro: write_etc_common
+  append: true
+  condition: and not proc.pname in (falco_binaries)
+```
+To make `falco` use this file, we'd need to tell Falco where to find these new rules. We can do that by editing the Helm `values.yaml` file again to make Falco aware of our custom rules file:
+
+```
+falco:
+  ...
+  rules_file:
+  - /etc/falco/falco_rules.yaml
+  - /etc/falco/falco_rules.local.yaml
+  - /etc/falco/rules.d
+  - /etc/falco/custom_falco_rules.yaml   <-- Add this line
```

Comment:
wdyt to remove that part and explain to use `--rulesfiles-dir` parameter of falcoctl to target `/etc/falco/rules.d` as folder?

Reply:
I removed it since that was messing with the rules-files:0. It's not possible to set the directory per artifact.